net-im/skypeforlinux/skypeforlinux-8.90.0.407-r1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MULTILIB_COMPAT=( abi_x86_64 )
+
+inherit chromium-2 desktop pax-utils rpm multilib-build xdg
+
+DESCRIPTION="Instant messaging client, with support for audio and video"
+HOMEPAGE="https://www.skype.com/"
+SRC_URI="https://repo.skype.com/rpm/stable/${PN}_${PV}-1.x86_64.rpm"
+S="${WORKDIR}"
+
+LICENSE="Skype-TOS MIT MIT-with-advertising BSD-1 BSD-2 BSD Apache-2.0 Boost-1.0 ISC CC-BY-SA-3.0 CC0-1.0 openssl ZLIB APSL-2 icu Artistic-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+QA_PREBUILT="*"
+RESTRICT="mirror bindist strip" #299368
+
+RDEPEND="
+	app-crypt/libsecret[${MULTILIB_USEDEP}]
+	|| (
+		>=app-accessibility/at-spi2-core-2.46.0:2[${MULTILIB_USEDEP}]
+		dev-libs/atk[${MULTILIB_USEDEP}]
+	)
+	dev-libs/expat[${MULTILIB_USEDEP}]
+	dev-libs/glib:2[${MULTILIB_USEDEP}]
+	dev-libs/nspr[${MULTILIB_USEDEP}]
+	dev-libs/nss[${MULTILIB_USEDEP}]
+	media-libs/alsa-lib[${MULTILIB_USEDEP}]
+	media-libs/fontconfig:1.0[${MULTILIB_USEDEP}]
+	media-libs/freetype:2[${MULTILIB_USEDEP}]
+	media-libs/libv4l[${MULTILIB_USEDEP}]
+	net-print/cups[${MULTILIB_USEDEP}]
+	sys-apps/dbus[${MULTILIB_USEDEP}]
+	sys-devel/gcc[cxx]
+	sys-libs/glibc
```

Comment:
I'm not sure calling gcc and glibc is required anymore but better to ask to someone more qualified

Reply:
Just leave it in. gcc is needed for libstdc++ here and glibc is needed as it won't work on musl.